ansible/playbooks/roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -10,7 +10,7 @@ node:
     - address: {{ hostvars[inventory_hostname].advertised_ip }}
       port: {{ redpanda_kafka_port }}
     advertised_rpc_api:
-    - address: {{ hostvars[inventory_hostname].advertised_ip }}
+      address: {{ hostvars[inventory_hostname].advertised_ip }}
```

Comment:
This resulted in an error due to the fact that `advertised_rpc_api` (unlike other listeners) isn't a list.